frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue

```diff
@@ -51,7 +52,7 @@
       <section>
         <h1 class="mini-composer__state-views__title">
           State Vector
-        </h1>
+        </h1>rm 
```

Comment:
I believe this wasn't meant to be included:

```suggestion
        </h1>
```